aerogear/keycloak-operator
# [WIP] Create shared service plan when keycloak is provisioned

## Description
Create a shared service plan once a Keycloak instance has finished provisioning. 

## Progress
- [x] When Keycloak has been marked as provisioned, create a shared service plan
- [x] Once a shared service plan has been created, mark the Keycloak resource's status as complete.

## Additional Notes
Related JIRA - https://issues.jboss.org/browse/AEROGEAR-7684

## Verification Steps

1. Run the managed-service-broker and the Keycloak operator in the same namespace.
2. Create a shared service custom resource
3. Verify that the SharedService resource is marked as `provisioning`
4. Verify that a SharedServicePlan resource is created once a Keycloak service instance has finished provisioning.
5. Verify that the SharedService and Keycloak resource is marked as `complete` once a SharedServicePlan resource is created.
6. Verify that the managed-service-broker picks up this SharedServicePlan
7. Verify that the plan is shown in the service catalog.
